### PR TITLE
Add instance: ask a few questions.

### PR DESCRIPTION
**Pull request by dalf**
Fix https://github.com/dalf/searx-stats2/issues/20 
Related to https://github.com/dalf/searx-instances/issues/23

### CODE REVIEW

**Comment by dalf** [comment]
request for comments
@asciimoo @unixfox @return42 @Pofilo @Caribpa

File: .github/ISSUE_TEMPLATE/add-instance.md [hunk 1]
```diff
@@ -6,9 +6,20 @@ labels: instance add, instance
...
+* [ ] I agree searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
+* [ ] My instance runs unmodified searx source code.
 
 <!--
-Replace <searx instance url> in the title by the URL.
+If you don't agree searx.space to check your instance, currently it is not possible to add your instance to the list.
+
+searx.space source code: https://github.com/dalf/searx-stats2
+
+If you have modified searx, please provide an URL to the source code, either here or in the theme of your instance.
```
Comment (Caribpa) [inline .github/ISSUE_TEMPLATE/add-instance.md]: According to the AGPLv3 those who modify the source code must have a public URL in their site to their full code. Thus I'd suggest to change this text to:
```
If you have modified searx, please provide an URL to the source code and make sure that the same URL is publicly accessible in your instance's site.
```

**Comment by return42** [comment]
I do not have any special meaning about .. since checking is elementary for a stat and user came to add there instance to that stat I do not see the need for a special agreement.  Are there really users having such problems with?

Anyway the comments you made in the patch are clarifying things more up .. you got my votes up :)

File: .github/ISSUE_TEMPLATE/add-instance.md [hunk 2]
```diff
@@ -6,9 +6,20 @@ labels: instance add, instance
 assignees: ''
 
 ---
+<!-- please answer the following questions (replace [ ] by [x] to say yes) -->
+* [ ] It is my instance.
+* [ ] I agree searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
```
Comment (Pofilo) [inline .github/ISSUE_TEMPLATE/add-instance.md]: `I agree` -> `I give the right to`

File: .github/ISSUE_TEMPLATE/add-instance.md (hunk 1 above)
Comment (dalf) [reply .github/ISSUE_TEMPLATE/add-instance.md]: I removed the question "My instance runs unmodified searx source code.". It seems an yes answer is mandatory to get the instance accepted.

About LGPL: I think it should be solve in the searx project, but a reminder here won't hurt, so I added the text you have suggested.

**Pull request merged**